app/scenes/channel/channel_post_list/channel_post_list.js

```diff
@@ -16,12 +16,17 @@ export default class ChannelPostList extends PureComponent {
         posts: PropTypes.array.isRequired
     };
 
+    state = {
+        lastViewedAt: this.props.myMember.last_viewed_at
+    };
+
     componentDidMount() {
         this.props.actions.loadPostsIfNecessary(this.props.channel);
     }
 
     componentWillReceiveProps(nextProps) {
         if (this.props.channel.id !== nextProps.channel.id) {
+            this.setState({lastViewedAt: nextProps.myMember.last_viewed_at});
```

Comment:
@enahum Yeah I was wondering about that possibility. Well because we're using the `getCurrentChannelMembership`, `myMember` will at least be an empty object if it isn't set. So `lastViewedAt` will be undefined. I made sure to check for its presence in `post_utils.js`, so it shouldn't cause a failure. The divider just won't be displayed if the timestamp isn't present. I'll test that to confirm.